be/src/formats/orc/apache-orc/c++/src/Reader.cc

```diff
@@ -337,17 +337,40 @@ void RowReaderImpl::loadStripeIndex() {
 
     // obtain row indexes for selected columns
     uint64_t offset = currentStripeInfo.offset();
+
+    static const uint64_t MAX_ONE_READ_ROW_INDEX_SIZE = 1024 * 1024;
```

Comment:
👍Could you annotate it

Reply:
Can we use `BufferedInputStream` to wrap around file for buffer read automatically?

Reply:
SeekableFileInputStream has already done this job. And it's in orc module.